### ISSUE TITLE
1440p CSS and settings button issue

### ISSUE THREAD

**Issue by KenanDK666** [dojwun/qb-inventory#21]
I'm having some issues with css layout on higher resolutions and the settings button not working.

![image](https://user-images.githubusercontent.com/61769938/145513186-5d43dc19-490d-469a-ac56-6e11695dd99f.png)

![image](https://user-images.githubusercontent.com/61769938/145513225-f98c3a0e-4032-464f-a857-e0bf108a9c38.png)

**Comment by dojwun** [comment]
unfortunately, i have no way to test 1440p but if someone that has access to 1440p or has a resolution fix there isnt much i can do

**Comment by OrionNetworks** [comment]
@dojwun if you DM zunq#0506 I will help you on discord with testing 1440p resolution

**Comment by dojwun** [comment]
@OrionNetworks sounds good i will get in contact with you later today

**Comment by dojwun** [comment]
@KenanDK666 ive also added a config for "⚙️" button, i set the button up to use lj-menu right now but you can change or disable it

**Comment by KenanDK666** [comment]
@dojwun awesome, i've changed my hud to lj-hud recently so that is just perfect.

**Comment by denisjecar1** [comment]
you can test it in chrome browser with any resolution with developer tools, the html/css is very broken, you are using % and combining with vh, no parent divs etc...

**Comment by dojwun** [comment]
yeah im not gona go through the whole inv "% and vh" for a resolution i dont use... feel free to fix it for other resolutions @denisjecar1

**Comment by denisjecar1** [comment]
you cant force players to use fullhd resolution only for UI to work correctly, any design made in 2021 should be responsive as fuck. I actually rewrote the whole css/html but I had to do it from scratch, there's no way to fix ur css when you are using hardcoded pixels and combining vh with %, yet alone hardcoding div tags in javascript. This is not how you build websites. Im not writing this to make you feel bad or anything, I meant it as a good advice, hope you take it that way

**Comment by dojwun** [comment]
@denisjecar1 you are taking this way too serious... calm down.. im not forcing anyone to do anything.. you needa move on if pixels are bothering you
